### DISCUSSION TITLE
Model training data and training source

### DISCUSSION THREAD

**Discussion by DanielWe2** [jomjol/AI-on-the-edge-device#1384] [Q&A]
I would like to look into training data and models.

I have read  the Wiki and looked into the different repositories and am still unsure were the actual models and training data is.

The main article seems to be:  https://github.com/jomjol/AI-on-the-edge-device/wiki/Learn-models-with-your-own-images

But for example it says:

```
dig-class100 / dig-cont models (digits)

Fork and checkout [neural-network-analog-needle-readout](https://github.com/jomjol/neural-network-analog-needle-readout).

```
dig-class100 / dig-cont models linking to analog-needle-readout that shouldn't be correct?

Then there is:  https://github.com/jomjol/neural-network-digital-counter-readout

That repository says that it is no longer used in favor of https://github.com/jomjol/neural-network-autotrain-digital-counter  but from the history in the readme and the last commits the first repo  seems the currently active?

My question is for the three main dig models (dig-class-11, dig-class-100, dig-cont (somewhere a dig-class10 is mentioned?) ) where is the latest version of the code to train the models and where is the training data:

### dig-class-11

**Source:**  :  https://github.com/jomjol/neural-network-digital-counter-readout  or https://github.com/jomjol/neural-network-autotrain-digital-counter (the later should be the newer from the docu but the first more recent)
**Data**:  In either of the two repositories?

### dig-class-100

**Source**: https://github.com/haverland/Tenth-of-step-of-a-meter-digit
**Data**:  Maybe also here:  https://github.com/haverland/Tenth-of-step-of-a-meter-digit   but it also references the other repository somewhere? I

### dig-class-cont

**Source**: I couldn't find the training source code at all?
**Data**:  Also https://github.com/haverland/Tenth-of-step-of-a-meter-digit  ?

I don't want to sound rude or something. I am really appreciating the work you guys put into it but it is really hard to understand :(

**Comment by jomjol** [comment]
For `dig-class-cont` I don't have published the training source and algorithm. This is not planned on a short term.

**Reply by DanielWe2** [reply]
Ok, thats fine. Could you clear up the confusion regarding the two repositorys for dig-class-11?

**Reply by jomjol** [reply]
Sure - the autotrain is an intermediate version with automated training action. Currently I don't use it. 
Sorry for the confusion within this. But this is rather for internal usage and currently to documented very well.
